install/kubernetes/quick-install.yaml

```diff
@@ -1,25 +1,27 @@
 ---
-# Source: cilium/charts/agent/templates/serviceaccount.yaml
+# Source: cilium/templates/cilium-agent-serviceaccount.yaml
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: cilium
   namespace: kube-system
 ---
-# Source: cilium/charts/operator/templates/serviceaccount.yaml
+# Source: cilium/templates/cilium-operator-serviceaccount.yaml
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: cilium-operator
   namespace: kube-system
 ---
-# Source: cilium/charts/config/templates/configmap.yaml
+# Source: cilium/templates/cilium-configmap.yaml
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: cilium-config
   namespace: kube-system
 data:
+  disable-envoy-version-check: "false"
```

Comment:
This option doesn't make sense to expose to users IMO.

Reply:
> Is there a way to avoid silently ignoring setting of values that are not used?
This is the default helm behavior. You can set any value you want, but the chart may or may not consume it.

> Is it possible to provide a message that X option was replaced by Y option?
Fully intend to add these notes to the documentation.

> Even a file that provides a list of the replacements would be useful so that we can point to it.
This can also be generated to include in docs, but it will be a very long table.

Unfortunately, there is no clean way to automatically take in a value as global.X and translate it to X. Users will need to follow the upgrade guide, which will clearly explain these changes. New users should not be impacted by this assuming the documentation reflects the updated values.

Reply:
removed